src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -222,7 +221,7 @@ func (cfg *Config) ApplyDefaults() {
 	}
 
 	if cfg.InstanceId == "" {
-		cfg.InstanceId = hex.EncodeToString(uuid.NodeID())
+		cfg.InstanceId = uuid.New()
```

Comment:
Yeah, the comment about '555' being used and initially seeing a few hits than none is definitely similar to what I've encountered. It sounds like if you don't vary the client id, future requests end up getting blocked.